src/inc/quic_platform_posix.h

```diff
@@ -988,73 +1025,171 @@ void
     _In_ CXPLAT_CQE* Cqe
     );
 typedef CXPLAT_EVENT_COMPLETION *CXPLAT_EVENT_COMPLETION_HANDLER;
+#define CXPLAT_USE_EVENT_BATCH_COMPLETION
+typedef
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+(CXPLAT_EVENT_BATCH_COMPLETION)(
+    _Inout_ CXPLAT_CQE** Cqes,
+    _Inout_ uint32_t* Count
+    );
+typedef CXPLAT_EVENT_BATCH_COMPLETION *CXPLAT_EVENT_BATCH_COMPLETION_HANDLER;
 typedef struct CXPLAT_SQE {
-    CXPLAT_EVENT_COMPLETION_HANDLER Completion;
+    CXPLAT_EVENT_COMPLETION_HANDLER ClassicCompletion;
+    CXPLAT_EVENT_BATCH_COMPLETION* Completion;
+    void* Context; // Review: create an encapsulating structure for this?
```

Comment:
Why is a void* insufficient and an encapsulating structure needed?

Reply:
Oh, I really should have fixed this - any context is typically stored in an encapsulating structure, e.g.:

```
CXPLAT_URING_SQE {
    CXPLAT_SQE Sqe;
    struct MY_URING_CONTEXT MyContext;
};
```

I'll update